triplea_maps.yaml

```diff
@@ -328,6 +328,30 @@
     <br>Leave feedback at the forums: http://triplea.sourceforge.net/mywiki/Community
     <br>or at the Depot: http://sites.google.com/site/tripleaerniebommel/
 - url: https://github.com/triplea-maps/big_world_2/releases/download/0.1/big_world_2.zip
```

Comment:
line 330 is for big world 2, it should be moved to *below* what you are inserting.

Instead you should have the following line here:
You're missing a line:
"- url: https://github.com/triplea-maps/big_world_1939/releases/download/0.5/big_world_1939.zip"